tests/app/api/services/test_abr.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+
+from app.api.services import abr_service
+from app.api.business.errors import AbrError
+import requests
+import mock
+from mock import patch
+
+
+class TestAbrService():
+
+    def mocked_find_business_by_abn(self):
+        data = '<ABR><response><stateCode>NSW</stateCode><postcode>2750</postcode>'\
+            '<organisationName>yay</organisationName></response></ABR>'
+        return data
+
+    def mocked_payload_exception(self):
+        data = '<ABR><response><exception><exceptionDescription>Search text is not a '\
+            'valid ABN or ACN</exceptionDescription><exceptionCode>WEBSERVICES</exceptionCode>'\
+            '</exception></response></ABR>'
+        return data
+
+    @mock.patch("app.api.services.abr_service.call_abr_api")
+    def test_parsed_data(self, mocked_find_business_by_abn):
+        """ test expected parsed data"""
```

Comment:
I think you can incorporate these comments into your test names.
```suggestion
    def test_abr_response_can_be_parsed(self, mocked_find_business_by_abn):
```